typescript/src/actions/solana/types.ts

```diff
@@ -84,4 +85,31 @@ export type AccountActions = {
    * ```
    */
   signTransaction: (options: Omit<SignTransactionOptions, "address">) => Promise<SignatureResult>;
+
+  /**
+   * Transfers SOL or SPL tokens between accounts
+   *
+   * @param {TransferOptions} options - Parameters for the transfer.
+   * @param {string} options.to - The base58 encoded Solana address of the destination account.
+   * @param {sol|usdc|string} options.token - The token to transfer ("sol" or "usdc"), or mint address of the SPL token to transfer.
+   * @param {bigint} options.amount - The amount to transfer, represented as an atomic unit (e.g. 0.01 * LAMPORTS_PER_SOL).
```

Comment:
I think the explanation here can be clearer:

> The amount to transfer in atomic units of the token. For example, 0.01 * LAMPORTS_PER_SOL would 
> transfer 0.01 SOL.

Reply:
Updated